schema.graphql

```diff
@@ -132,11 +132,12 @@ type CW20Transfer @entity {
   transaction: Transaction!
   block: Block!
 }
-  
+
 type Account @entity {
   # id is the address
   id: ID!
   nativeBalanceChanges: [NativeBalanceChange]! @derivedFrom(field: "account")
```

Comment:
Can we make this `cw20BalanceChanges`? I think that's a better fit with convention.

Reply:
Can we also try to make the type `Cw20BalanceChange` like we talked about?